index.js

```diff
@@ -179,6 +181,78 @@ class ServerlessS3Sync {
       });
   }
 
+  syncMetadata() {
+    const s3Sync = this.serverless.service.custom.s3Sync;
+    const cli = this.serverless.cli;
+    if (!Array.isArray(s3Sync)) {
+      cli.consoleLog(`${messagePrefix}${chalk.red('No configuration found')}`)
+      return Promise.resolve();
+    }
+    cli.consoleLog(`${messagePrefix}${chalk.yellow('Syncing metadata...')}`);
+    const servicePath = this.servicePath;
+    const promises = s3Sync.map( async (s) => {
+      let bucketPrefix = '';
+      if (s.hasOwnProperty('bucketPrefix')) {
+        bucketPrefix = s.bucketPrefix;
+      }
+      let acl = 'private';
+      if (s.hasOwnProperty('acl')) {
+        acl = s.acl;
+      }
+      if (!s.bucketName || !s.localDir) {
+        throw 'Invalid custom.s3Sync';
+      }
+      const localDir = [servicePath, s.localDir].join('/');
+      let filesToSync = [];
+      if(Array.isArray(s.params)) {
+        s.params.forEach((param) => {
+          const glob = Object.keys(param)[0];
+          let files = this.getLocalFiles(localDir, []);
+          minimatch.match(files, `${path.resolve(localDir)}/${glob}`, {matchBase: true}).forEach((match) => {
+            filesToSync.push({name: match, params: this.extractMetaParams(param)});
+          });
+        });
+      }
+      return filesToSync.forEach((file) => {
+        return new Promise((resolve) => {
+          let params = {
+            CopySource: file.name.replace(localDir, `${s.bucketName}${bucketPrefix == '' ? '' : bucketPrefix}/`),
+            Key: file.name.replace(localDir, ''),
+            Bucket: s.bucketName,
+            Metadata: file.params,
+            MetadataDirective: 'REPLACE'
+          };
```

Comment:
require `ACL` ?
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#copyObject-property

Reply:
acl now works the same as the sync operation.  if you don't set it, it will default to private.  If you set it then it will be what you desire.